app/src/main/java/com/github/polypoly/app/base/game/Player.kt

```diff
@@ -18,6 +18,7 @@ data class Player (
     val user: User = User(),
     private var balance: Int = 0,
     private var roundLost: Int? = null,
+    private val bill : Set<Transaction> = Transaction.values().toSet(),
```

Comment:
I don't think this is the best place to put this: as it is a turn-specific value it would better fit in Game as this notion of turn is not really in the conceptual space of the player: it doesn't know in which turn it is, not wether is should clear its bill or not, whereas `Game` does know that and is responsible for game turns' consistency